README.md

```diff
@@ -30,25 +30,31 @@
     * Fedora: `oc cluster up --image="docker.io/openshift/origin"`
     * Mac OSX: Follow the Minishift [Getting Started Guide](https://docs.openshift.org/latest/minishift/getting-started/index.html)
     * Note: Startup output will contain the URL to the web console for your openshift cluster, save this for later
-  * Login to the OpenShift cluster as admin:
+  * Login to the OpenShift cluster as system:admin:
     * `oc login -u system:admin`
-  * Grant admin rights to login to the web console
+  * Create an "admin" account with cluster-admin role which you can use to login to the [WebUI](https://localhost:8443) or with oc:
```

Comment:
You need to oc login as this regular account so the whoami can give you a token, that does not work for system:admin.